docs/framework/debug-trace-profile/making-an-image-easier-to-debug.md

```diff
@@ -1,55 +1,54 @@
+
+This JIT configuration has two aspects:
+
+- You can request the JIT-compiler to generate tracking information. This makes it possible for the debugger to match up a chain of MSIL with its machine code counterpart, and to track where local variables and function arguments are stored. Starting with the .NET Framework version 2.0, the JIT compiler always generates tracking information, so there is no need to request it.
+
+- You can request the JIT-compiler to not optimize the resulting machine code.
+
+Normally, the compiler that generates the MSIL sets these JIT-compiler options appropriately, based upon the IDE switches or command-line options you specify, for example, /**Od**.
+
+In some cases, you might want to change the behavior of the JIT compiler so that the machine code it generates is easier to debug. For example, you might want to generate JIT tracking information for a retail build or control optimization. You can do so with an initialization (.ini) file.
+
+For example, if the assembly you want to debug is called *MyApp.exe*, then you can create a text file named *MyApp.ini*, in the same folder as *MyApp.exe*, which contains these three lines:
+
+```txt
+[.NET Framework Debugging Control]
+GenerateTrackingInfo=1
+AllowOptimize=0
+```
+
+You can set the value of each option to 0 or 1, and any absent option defaults to 0. Setting `GenerateTrackingInfo` to 1 and `AllowOptimize` to 0 provides the easiest debugging.
+
+Starting with the .NET Framework version 2.0, the JIT compiler always generates tracking information regardless of the value for `GenerateTrackingInfo`; however, the `AllowOptimize` value still has an effect. When using the [Ngen.exe (Native Image Generator)](../../../docs/framework/tools/ngen-exe-native-image-generator.md) to precompile the native image without optimization, the .ini file must be present in the target folder with `AllowOptimize=0` when Ngen.exe executes. If you have precompiled an assembly without optimization, you must remove the precompiled code using the NGen.exe **/uninstall** option before rerunning Ngen.exe to precompile the code as optimized. If the .ini file isn't present in the folder, by default Ngen.exe precompiles the code as optimized.
+
+The <xref:System.Diagnostics.DebuggableAttribute?displayProperty=nameWithType> controls the settings for an assembly. **DebuggableAttribute** includes two fields that record the settings for whether the JIT compiler should optimize, and/or generate tracking information. Starting with the .NET Framework version 2.0, the JIT compiler always generates tracking information.
```

Comment:
that record the settings for whether --> that control whether
no comma after optimize